tool/lib/test/unit/testcase.rb

```diff
@@ -256,8 +256,14 @@ def self.inherited klass # :nodoc:
         super
       end
 
-      def self.test_order # :nodoc:
-        :sorted
+      @test_order = :sorted
+
+      class << self
+        attr_accessor :test_order
```

Comment:
```suggestion
        attr_writer :test_order
```

The `reader` is overridden soon.